src/omnicore/version.h

```diff
@@ -15,13 +15,13 @@
 #define OMNICORE_VERSION_MILESTONE   0
 
 // Increase with every consensus affecting change
-#define OMNICORE_VERSION_MAJOR       10
+#define OMNICORE_VERSION_MAJOR       0
```

Comment:
This is what I think, too, but I'm a bit confused nevertheless, hehe. See here:

https://github.com/OmniLayer/omnicore/blob/30fdde4833ac20c130d62e9d560b3b0d70ee8744/src/omnicore/version.cpp#L37-L53